src/jit/morph.cpp

```diff
@@ -7243,7 +7244,7 @@ bool Compiler::fgCanFastTailCall(GenTreeCall* callee)
     }
 
     // Callee has a >8 and <=16 byte struct and arguments that has to go on the stack. Do not fastTailCall.
-    if (calleeStackSize > 0 && hasTwoSlotSizedStruct)
+    if (hasStackArgs > 0 && hasTwoSlotSizedStruct)
```

Comment:
I think this should just be `if (hasStackArgs && hasTwoSlotSizedStruct)`

Reply:
Oops thanks :)